nbextensions/usability/spellchecker/README.md

```diff
@@ -0,0 +1,39 @@
+This extension provides a CodeMirror overlay mode to highlight incorrectly
+spelled words in Markdown and Raw cells:
+
+![screenshot.png](./screenshot.png)
+
+It was inspired in part by
+[NextStepWebs/codemirror-spell-checker](https://github.com/NextStepWebs/codemirror-spell-checker/blob/78773ebdd6c8cf8acd043342023636ae345ca0f3/src/js/spell-checker.js)
+at the
+[suggestion](https://github.com/ipython-contrib/IPython-notebook-extensions/issues/521)
+of [@JanSchulz](https://github.com/JanSchulz).
```

Comment:
Ah, you're welcome. Besides, I like to keep a history of where ideas come from, sometimes it's handy to remind yourself when you want to come back to something after not looking at it for some time!